torchao/prototype/autoround/core.py

```diff
@@ -94,7 +98,9 @@ def prepare_model_for_applying_auto_round_(
         group_size (int, optional): The group size for quantization. Defaults to 128.
         iters (int, optional): The number of iterations for optimization. Defaults to 200.
         use_optimized_layer_output (bool, optional): Whether to use optimized layer output. Defaults to False.
-        device (Optional[torch.types.Device], optional): The device to use for accelrating optimization and calibration.
+        gradient_accumulate_steps (Optional[int]): The number of gradient accumulation steps. Defaults to 1.
```

Comment:
maybe a little bit more docs for the context of gradient accumulation might be helpful, e.g. when this is used in autoround etc.

Reply:
Thanks, updated.